font/LICENSE.txt

```diff
@@ -0,0 +1,12 @@
+Font license info
+
+
+## Font Awesome
+
+   Copyright (C) 2016 by Dave Gandy
+
+   Author:    Dave Gandy
+   License:   SIL ()
```

Comment:
Please add either the link to the license document or the text of the license itself.

Reply:
OK, I added a link to the license.

Fontello says on https://github.com/fontello/fontello/wiki/What-about-license%3F:

> If you place content of fontello's archive somewhere with your source code, that's enough. See, example for fontello itself.